intern/remote/remote.py

```diff
@@ -312,4 +312,22 @@ def mesh(self, resource, resolution,
         mesh = self._mesh.create(
             volume, x_range, y_range, z_range, time_range, id_list, voxel_unit, voxel_size,
             simp_fact, max_simplification_error, normals)
-        return mesh
\ No newline at end of file
+        return mesh
+
+    def get_cuboids_from_id(self, resource, resolution, id):
+        """Get corners for cuboids that belong to a specific ID. 
+
+        All returned corners are cuboid aligned.
```

Comment:
I think we should say _which_ corner is returned. IIRC, it's the corner closest to the origin?